Common/MemoryUtil.cpp

```diff
@@ -325,17 +324,16 @@ void ProtectMemoryPages(const void* ptr, size_t size, uint32_t memProtFlags) {
 #endif
 }
 
-// Hardcoded in the header. This is a way to check it dynamically in case it ever becomes necessary.
-/*
+int GetMemoryProtectPageSize() {
 #ifdef _WIN32
+	// This is 4096 on all Windows platforms we care about. 8k on Itanium but meh.
+	return 4096;
+	// For reference, here's how to check:
+	// if (sys_info.dwPageSize == 0)
+	//   GetSystemInfo(&sys_info);
```

Comment:
FWIW, we're doing that already: https://github.com/hrydgard/ppsspp/blob/1e4b77fb49aa698f99eafd54ee3924b37a47310b/Common/MemoryUtil.cpp#L43

-[Unknown]